internal/app/machined/pkg/controllers/block/volume_config.go

```diff
@@ -232,7 +232,8 @@ func (ctrl *VolumeConfigController) manageEphemeral(config cfg.Config) func(vc *
 		}
 
 		vc.TypedSpec().Mount = block.MountSpec{
-			TargetPath: constants.EphemeralMountPoint,
+			TargetPath:   constants.EphemeralMountPoint,
+			SelinuxLabel: constants.EphemeralSelinuxLabel,
```

Comment:
We need to relabel ephemeral and other block-backed FSs, and also run update tests

Reply:
As discussed will do this later, for now SELinux will be only for new installs. Later we'll add support to the update flow to take care of it.